### DISCUSSION TITLE
[Idea] 全文搜索与 CMD K

### DISCUSSION THREAD

**Discussion by arvinxx** [lobehub/lobehub#583] [Ideas]
全文搜索可以考虑使用 [flex-search](https://github.com/nextapps-de/flexsearch) 实现，性能炸裂：

https://nextapps-de.github.io/flexsearch/bench/

<img width="1083" alt="image" src="https://github.com/lobehub/lobe-chat/assets/28616219/5c6e3407-e3e2-45d0-9169-b2191575b47b">

`CMD` + `K`

- https://cmdk.paco.me/
- https://react-cmdk.com/

**Comment by canisminor1990** [comment]
吓到
